admin/views/partials/_content.ejs

```diff
@@ -1,8 +1,8 @@
-<main id="content" role="main">
+<div id="content" role="main">
```

Comment:
This will affect the main container for every page and is not GDS compliant.

I would suggest changing it back to a `<main>`
 element and applying a print style sheet that would override the style that are causing the extra page to be printed.

Examples:

* https://github.com/UKGovernmentBEIS/rifs-frontend-play/blob/master/src/main/assets/stylesheets/application-print.scss
* https://github.com/alphagov/government-frontend/blob/master/app/assets/stylesheets/print.scss

Although after a cursory look at the stylesheets above, it doesn't look like they are dealing with extra blank pages produced when printing.  

If you can get to the bottom of it and find any evidence that it is caused by the GDS styles then we should raise an issue to at least get the documentation updated to point this out, possibly including our fix if it will work for everybody.